utils/dataloaders.py

```diff
@@ -29,11 +29,11 @@ def get_dataloaders(batch_size, dataset):
         "celeba": get_celeba_dataloader
     }
     if dataset in dataset_options:
-        return dataset_options[dataset](batch_size=batch_size)
+        return dataset_options[dataset](batch_size=batch_size, shuffle=shuffle)
     else:
         raise Exception("{} is not valid. Please enter a valid dataset".format(dataset))
 
-def get_mnist_dataloaders(batch_size=128,
+def get_mnist_dataloaders(batch_size=128, shuffle=False,
```

Comment:
Here you are setting shuffle=False in the get_mnist_dataloaders function call, but in the call to DataLoader below you are resetting it to True. So the argument in get_mnist_dataloaders can't affect the call to DataLoader. I'm assuming this was not the intention?

Reply:
This just sets the default to ```False```. Anything passed to the function will override the default ```shuffle=False``` setting.

Reply:
Yes, I'm aware it's just the default setting. But inside the function `get_mnist_dataloaders` you are calling the function `DataLoader` of which one of the arguments is `shuffle=True`. My question is, should the the internal function `DataLoader` always have the argument  `shuffle=True`? Or should it inherit the boolean value of the parent function?

Reply:
Oh I understand. Yeah I think that you are correct. I'll make the required change. Thanks.

Reply:
Cool. FYI -  this error occurs in a number of the functions in this file.